easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'TetGen'
+version = '1.5.0'
+
+homepage = 'http://wias-berlin.de/software/index.jsp?id=TetGen'
+description = """ A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator """
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['https://github.com/ufz/tetgen/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['791dfc6d8bf7bf2496f10da1d6d649db5e2fb12267d7911e21f4f335f374698e']
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+    ('binutils', '2.28'),
+]
+
+separate_build_dir = True
+
+skipsteps = ['install']
+
+postinstallcmds = [
+    'mkdir -p %(installdir)s/bin/ && cp tetgen %(installdir)s/bin/',
+    'mkdir -p %(installdir)s/lib/ && cp libtet.a %(installdir)s/lib/',
```

Comment:
Hmm, `make install` doesn't work as expected? That's a bit strange with `CMake` being used, no?

Reply:
"make install" is not defined. You know a better way to handle this?

Reply:
Isn't `cmake` supposed to make sure it *is* defined in the generated `Makefile`? @wpoely86?

Reply:
yes and no. You still need to tell CMake what it is supposed to install and where. This `CMakeFile.txt` is very minimal and only has the instructions to build it. It's this or we patch the `CMakeFile.txt`.

Reply:
Ugh, ok fine, anything is better than patching the `CMakeFile.txt`. ;-)